### ISSUE TITLE
WCAG 1.3.1 A: Current page shown with presentation alone (Issue 10)	

### ISSUE THREAD

**Issue by CJE001** [civictheme/uikit#376]
### Summary

Via Vision Australia assessment: August 2024

Impact: low

Note: DTA have a 90 day remediation period to address the identified issues within the audit, all issues must be resolved to obtain WCAG 2.2 certification for digital.gov.au.

### Steps to reproduce

Global – Main navigation
[Contact digital.gov.au beta | digital.gov.au](https://www.digital.gov.au/contact) (https://www.digital.gov.au/contact)

### Observed outcome

Within the main navigation, the current location is clear visually through a thick underline displayed on the current link; however, this information is not available programmatically.

![Picture38](https://github.com/user-attachments/assets/2bf0a42b-a507-4998-842a-0965a2f62a45)

### Code used
**HTML**
```
<li class="ct-menu__item ct-menu__item--level-0 ct-menu__item--active-trail ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/contact" title="Contact">
Contact
</a>
</li>
```
**CSS**
```
.ct-navigation.ct-navigation--drawer .ct-navigation__items .ct-navigation__menu .ct-menu__item--level-0 {
border-bottom: solid .25rem;
}
```
### Why this matters
Screen reader users may have difficulty determining their location within the navigation. 

### Expected outcome

Programmatically provide information on the current page. This can be achieved via the aria-current attribute on the current link within the navigation list. As this list represents pages, set the value of the attribute to "page".

The position of this attribute should be dynamically updated as the current page changes.

### Code example

```
<li class="ct-menu__item ct-menu__item--level-0 ct-menu__item--active-trail ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/contact" aria-current="page">
Contact
</a>
</li>
```

### Additional example
Global issue – Side navigation
[Observatory news: February 2024 | digital.gov.au](https://www.digital.gov.au/initiatives/observatory/observatory-news/observatory-news-february) (https://www.digital.gov.au/initiatives/observatory/observatory-news/observatory-news-february)

### **Description**

Within the side navigation, the current location is clear visually through a change in background colour. However, this information is not available programmatically.

![Picture37](https://github.com/user-attachments/assets/278b9db3-6a86-4055-a4d0-22c739207b0c)
 
### Code used
**HMTL**
```
<li class="ct-menu__item ct-menu__item--level-2 ct-menu__item--active-trail ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/initiatives/observatory/observatory-news/observatory-news-february" title="February newsletter" tabindex="-1">
February newsletter
</a>
</li>
```
**CSS**
```
.ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item.ct-menu__item--active-trail>.ct-menu__item__link, .ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item.ct-menu__item--active-trail>.ct-menu__item__link:active, .ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item.ct-menu__item--active-trail>.ct-menu__item__link:visited {
background-color: #dedede;
border-color: var(--ct-side-navigation-light-link-child-active-border-color);
color: var(--ct-side-navigation-light-link-child-active-color);
}
```
### Recommendation
Please refer to the recommendation in the primary example.

Additionally, as the links that trigger the disclosures have also been visually styled to convey their state, the aria-current attribute with a value of "location" or "true" may be added to these elements, however, as they already have the aria-expanded attribute, this should be sufficient to convey equivalent information to screen reader users. 

Furthermore, the current location should not rely on a change of colour alone to visually convey information. Please refer to Issue 13 for more information. 

### Code example

```
<li class="ct-menu__item ct-menu__item--level-2 ct-menu__item--active-trail ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/initiatives/observatory/observatory-news/observatory-news-february" aria-current="page">
February newsletter
</a>
</li>
```
